### PR TITLE
Pumpio does connect now again.

### PR DESCRIPTION
**Pull request by annando**
See post here: https://forum.friendi.ca/display/0b6b25a8945b8636bac842e008892922

### CODE REVIEW

File: pumpio/pumpio.php [hunk 1]
```diff
@@ -361,20 +362,17 @@ function pumpio_settings_post(App $a, array &$b)
...
+			PConfig::set(local_user(), 'pumpio', 'host'           , $host);
+			PConfig::set(local_user(), 'pumpio', 'user'           , $user);
+			PConfig::set(local_user(), 'pumpio', 'public'         , defaults($_POST, 'pumpio_public', false));
+			PConfig::set(local_user(), 'pumpio', 'mirror'         , defaults($_POST, 'pumpio_mirror', false));
+			PConfig::set(local_user(), 'pumpio', 'post_by_default', defaults($_POST, 'pumpio_bydefault', false));
+
+			if (!empty($_POST['pumpio_mirror'])) {
+				PConfig::delete(local_user(), 'pumpio', 'lastdate');
 			}
-			//header("Location: ".$a->get_baseurl()."/pumpio/connect");
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Please remove commented out code.

File: pumpio/pumpio.php [hunk 2]
```diff
@@ -139,7 +139,8 @@ function pumpio_registerclient(App $a, $host)
 function pumpio_connect(App $a)
 {
 	// Start a session.  This is necessary to hold on to  a few keys the callback script will also need
-	session_start();
+	// Currently disabled, since a session is already running
+	//session_start();
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Please remove commented out code.

File: pumpio/pumpio.php (hunk 1 above)
Comment (annando) [reply pumpio/pumpio.php]: I removed that code - that's why the line is red :-)

Comment (MrPetovan) [reply pumpio/pumpio.php]: *facepalm*

**Pull request merged**